JSDemos/Demos/PivotGrid/DrillDown/description.md

```diff
@@ -1 +1,5 @@
-Drill-down support within the PivotGrid component allows you to&nbsp;retrieve the list of&nbsp;records used to&nbsp;calculate a&nbsp;specific summary value. To&nbsp;see drill downs in&nbsp;action, click a&nbsp;summary cell and view the records that formed the basis of&nbsp;the summary value.
\ No newline at end of file
+PivotGrid supports the drill-down operation that allows you to retrieve individual facts (records) used to calculate a specific summary value.
```

Comment:
What do you mean by "facts"? Is there a more specific term we can use?

Reply:
Hello. This is a basic term that is familiar to people who work with data (analysts, data engineers, etc). You can find it in almost any related article: https://doc.sitecore.com/en/developers/90/sitecore-experience-platform/facts-and-dimensions.html

Reply:
Okay, thanks for the clarification. :)